jsonrpc2.go

```diff
@@ -365,6 +365,11 @@ func (c *Conn) Call(ctx context.Context, method string, params, result interface
 	if err != nil {
 		return err
 	}
+	defer func() {
+		for range call.done {
+			// drain
+		}
+	}()
```

Comment:
I don't understand. On timeout, `call.done` will still get some value and be closed eventually. That's exactly the goroutine leak that we see.